### ISSUE TITLE
Possibility of replacing in property name

### ISSUE THREAD

**Issue by CKGrafico** [gridonic/postcss-replace#156]
I've this example of code where I replace the ## or $ to use simple vars and achieve this:

```
.u-color-basic-dark{
      color:#9c9c9c;
    }
```

my problem is that this code:
```
@each ##key, ##value in entries(##map) {
    &-##(prop)-##(name)-##(key) {
      ##(prop): ##value;
    }
  }
```

generates this code
```
.u-color-basic-darker{
      ##(prop):#585858;
    }
```

is it possible to use replace in the property name?

**Comment by dschenk** [comment]
Hi @CKGrafico, thanks for the issue and the fix! I'll take a look this week and merge it if everything works as expected

**Comment by dschenk** [comment]
@CKGrafico: I merged your commit, added another test, did some clean-up work and published a new version: 1.1.3

**Comment by CKGrafico** [comment]
Thanks for this awesome lib :D
